app/models/invoice.rb

```diff
@@ -0,0 +1,6 @@
+class Invoice < ApplicationRecord
+  belongs_to :customer
+  # has_many :transactions
```

Comment:
These are commented out as they will pass when our changes are merged with the group!